scripts/devnet-algo/rotate_dbg.sh

```diff
@@ -0,0 +1,10 @@
+cd /dbg
+for i in $(ls | grep [[:digit:]]-  | awk -F- '{print $1}' | uniq); do 
```

Comment:
I believe that `uniq` must occur after `sort`, because `ls` is not guaranteed to return sorted

Reply:
The order here doesn't really matter actually, here I'm only concerned about the section groups (15-,16-,17-...).
I could still sort them if you'd like.